### PR TITLE
[TeamQuest] Add more methods to TeamQuestGroup, add tests.

### PR DESCRIPTION
**Pull request by razvan3895**

### CODE REVIEW

File: wouso/games/teamquest/tests.py [hunk 1]
```diff
@@ -15,3 +15,51 @@ def setUp(self):
 	def test_is_head(self):
 		group = TeamQuestGroup.create(head=self.special_user, name='_test_group')
 		self.assertTrue(self.special_user.is_head())
```
Comment (iulianR) [inline wouso/games/teamquest/tests.py]: It should be team_quest_user or just user

File: wouso/games/teamquest/tests.py [hunk 2]
```diff
@@ -15,3 +15,51 @@ def setUp(self):
...
+
+	def test_is_added(self):
+		p = User.objects.create(username="_test_head")
+		p = p.get_profile().get_extension(TeamQuestUser)
+		group = TeamQuestGroup.create(head=p, name='_test_group')
+		group.add(self.special_user)
```
Comment (iulianR) [inline wouso/games/teamquest/tests.py]: special_user -> user

Rename it everywhere in file

File: wouso/games/teamquest/models.py [hunk 3]
```diff
@@ -23,20 +23,57 @@ class TeamQuestGame(Game):
...
+    def __unicode__(self):
+        return u"%s [%d]" % (self.name, self.users.count())
+
 
 class TeamQuestStatus(models.Model):
 	pass
 
 class TeamQuestInvitation(models.Model):
-	pass
+	from_group = models.ForeignKey('TeamQuestGroup')
```
Comment (iulianR) [inline wouso/games/teamquest/models.py]: Indentation (probably tab instead of spaces)

File: wouso/games/teamquest/models.py [hunk 4]
```diff
@@ -23,20 +23,57 @@ class TeamQuestGame(Game):
...
+    to_user = models.ForeignKey('TeamQuestUser')
+
+    def __unicode__(self):
+        return u"Invitation from %s to %s" % (self.from_group.head, self.to_user)
+
 
 class TeamQuestInvitationRequest(models.Model):
-	pass
+	to_group = models.ForeignKey('TeamQuestGroup')
```
Comment (iulianR) [inline wouso/games/teamquest/models.py]: Indentation (probably tab instead of spaces)

**Comment by iulianR** [comment]
Fix indentation everywhere in the tests.py file.

Also, you should add docstrings [1] to methods.
[1] https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring

**Comment by razvan3895** [comment]
Should be good now.

File: wouso/games/teamquest/models.py [hunk 5]
```diff
@@ -14,29 +14,66 @@ def is_head(self):
...
+        self.head = user
+        self.head.save()
+
+    def __unicode__(self):
+        return u"%s [%d]" % (self.name, self.users.count())
+
 
 class TeamQuestStatus(models.Model):
-	pass
+    pass
```
Comment (iulianR) [inline wouso/games/teamquest/models.py]: 2 blank lines between classes (PEP8)

File: wouso/games/teamquest/models.py [hunk 6]
```diff
@@ -14,29 +14,66 @@ def is_head(self):
...
 
 class TeamQuestInvitation(models.Model):
-	pass
+    from_group = models.ForeignKey('TeamQuestGroup', null=True, blank=False)
+    to_user = models.ForeignKey('TeamQuestUser', null=True, blank=False)
+
+    def __unicode__(self):
+        return u"Invitation from %s to %s" % (self.from_group.head, self.to_user)
+
```
Comment (iulianR) [inline wouso/games/teamquest/models.py]: 2 blank lines between classes (PEP8)

**Pull request merged**